packages/contracts/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Added a `VoteMode` enumeration to specify if the vote should be conducted in `Standard`, `EarlyExecution`, or `VoteReplacement` mode.
```

Comment:
What are your feelings about this enumerations growing in the future? Depending on how we define it it may turn into an unstructured/inconsistent list. But the alternative is allowing to pass flags which may make no sense together

Reply:
What problem do you see? I find this enumeration being an improvement.
`bool` in solidity are stored as `uint8`.
 Passing multiple`bool`s increases storage and runtime gas costs and complexity (because of more complicated if / else conditions).
 I doubt that we will introduce more voting modes in the future, but in case we do, we can put them in this enumeration.

Reply:
I also doubt we do, but in my past experience this enumeration ended up as a bitmask with tons of potential (unstructured) combinations, and it had to be maintained.

If you are confident, then let's resolve this comment